distro/superbuild/cmake/externals.cmake

```diff
@@ -224,9 +242,18 @@ endif()
 
 ###############################################################################
 # PythonQt
+set(PythonQt_REPO)
+set(PythonQt_TAG)
+if(${DD_QT_VERSION} VERSION_GREATER "4")
+  set(PythonQt_REPO https://github.com/patmarion/PythonQt.git)
+  set(PythonQt_TAG fix-compile-error-on-qt5.8)
```

Comment:
Sigh... commontk/PythonQt#50 (apparently I can create a PR of Pat's branch).

@sankhesh, does [patched-7](https://github.com/commontk/PythonQt/tree/patched-7) still work with Qt4?